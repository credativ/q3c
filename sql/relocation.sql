set search_path to tests,public;
select count(*) from test where q3c_radial_query(ra,dec,171.890000,-85.710000,0.1);
select count(*) from test where q3c_radial_query(ra,dec,45.360000,-8.020000,0.1);
select count(*) from test where q3c_radial_query(ra,dec,247.030000,50.850000,0.1);
select count(*) from test where q3c_radial_query(ra,dec,308.900000,42.260000,0.1);
