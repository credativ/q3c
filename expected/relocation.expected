SET
 count 
-------
     0
(1 row)

 count 
-------
     2
(1 row)

 count 
-------
     1
(1 row)

 count 
-------
     2
(1 row)

